server/src/connectionHandlers/onCompletion.ts

```diff
@@ -96,6 +99,16 @@ function getBitBakeCompletionItems (textDocumentPositionParams: TextDocumentPosi
       ]
     }
 
+    const variablesAllowedForLicenseCompletion = ['LICENSE']
+    const isVariableAllowedForLicenseCompletion = analyzer.isStringContentOfVariableAssignment(documentUri, wordPosition.line, wordPosition.character, variablesAllowedForLicenseCompletion)
+    if (isVariableAllowedForLicenseCompletion && recipeLocalFiles !== undefined && word !== null) {
+      const textDocument = analyzer.getAnalyzedDocument(documentUri)?.document
+      if (textDocument !== undefined) {
+        const rangeOfText = getRangeOfTextToReplace(textDocument, textDocumentPositionParams.position)
+        return getLicenseCompletionItems(rangeOfText)
```

Comment:
You should also add the operators with properly labeled category (`&, `|`).

Reply:
Possibly another PR if you want

Reply:
You mean stuff like
```
LICENSE = "GPL-2.0-or-later | BSD-2-Clause 0BSD"
```
?

So when the user is typing, `&` and `|` should also be included in the suggestions?

Also I see those a lot in BitBake code, but I don't see them in [BitBake documentation](https://docs.yoctoproject.org/bitbake/bitbake-user-manual/bitbake-user-manual-ref-variables.html#term-LICENSE) or [SPDX documentation](https://spdx.github.io/spdx-spec/v3.0/annexes/SPDX-license-expressions/). In SPDX documentation, I rather see `AND`, `OR` and `WITH`.

Reply:
Yes in bitbake, the operators are `&` and `|`, maybe the SPDX ones are also supported?
It could be nice to have suggestion for the operators appearing at the top of the list.

Reply:
Done.

I searched in Poky for occurrences of the SPDX operators with the approximate following regex: `license .?=.* (and|AND|or|OR|with|WITH)`. Could not find anything.